src/modules/fundingManager/depositVault/interfaces/IFM_DepositVault_v1.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+interface IFM_DepositVault_v1 {
+    //--------------------------------------------------------------------------
+    // Events
+
+    /// @notice Event emitted when a deposit takes place.
+    /// @param _from The address depositing tokens.
+    /// @param _amount The amount of tokens deposited.
+    event Deposit(address indexed _from, uint _amount);
```

Comment:
You might want to add here address indexed _to, to allow an account to deposit into another's "credit"

Reply:
Now that I removed `depositFor`, this is not applicable anymore, right?
I've removed it in [this commit](https://github.com/InverterNetwork/contracts/pull/646/commits/ef74cf6e825dc685712d95b1f9e3614da9c0eb20). The PR handling all fixes is #626.